src/main/kotlin/fr/social/gouv/agora/infrastructure/qag/repository/QagInfoRepositoryImpl.kt

```diff
@@ -28,6 +33,19 @@ class QagInfoRepositoryImpl(
         }
     }
 
+    override fun insertQagInfo(qagInserting: QagInserting): QagInsertionResult {
+        repeat(MAX_INSERT_RETRY_COUNT) {
+            mapper.toDto(qagInserting)?.let { qagDTO ->
+                if (!databaseRepository.existsById(qagDTO.id)) {
+                    databaseRepository.save(qagDTO)
+                    cacheRepository.insertQag(qagUUID = qagDTO.id, qagDTO = qagDTO)
```

Comment:
Est-ce qu'on ferait pas la modif dès maintenant ?

Reply:
of course!